homeassistant/components/overkiz/quality_scale.yaml

```diff
@@ -0,0 +1,68 @@
+rules:
```

Comment:
`Expected number of shower` is both a sensor and number?

Reply:
Yes, this might be legacy from before the number entity. Does this need to be cleaned up? (as this will be a breaking change).